_data/chaselundell/chaselundell.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Chase
+institution: Utah State University
+profile_pic: usucrest.jpg
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `usucrest.jpg` to `usuCrest.jpg ` in the markdown file.
2. Rename `usuCrest.jpg ` to `usucrest.jpg`.
I'd suggest you rename the file!